
	<launch>
		<arg name="calibration" default="$(find velodyne_pointcloud)/params/VLP16db.yaml"/>
		<arg name="max_range" default="130.0" />
		<arg name="min_range" default="0.9" />
		
		<node pkg="velodyne_stamped_points" type="velodyne_stamped_points_node" name="velodyne_stamped_points_node" output="screen" >
			<param name="calibration" value="$(arg calibration)"/>
			<param name="max_range" value="$(arg max_range)"/>
			<param name="min_range" value="$(arg min_range)"/>
<!--             <remap from="velodyne_packets" to="/velodyne16/velodyne_packets" />			 -->
		</node>
	</launch>


<!-- launch-prefix="gdb -ex run \-\-args"-->
